model/utils_test.go

```diff
@@ -18,18 +18,14 @@ import (
 func TestNewId(t *testing.T) {
 	for i := 0; i < 1000; i++ {
 		id := NewId()
-		if len(id) > 26 {
-			t.Fatal("ids shouldn't be longer than 26 chars")
-		}
+		require.LessOrEqual(t, len(id), 26, "ids shouldn't be longer than 26 chars")
 	}
 }
 
 func TestRandomString(t *testing.T) {
 	for i := 0; i < 1000; i++ {
 		r := NewRandomString(32)
-		if len(r) != 32 {
-			t.Fatal("should be 32 chars")
-		}
+		require.Equal(t, len(r), 32, "should be 32 chars")
```

Comment:
You can use `require.Len` assertion here and you can remove entirely the final string, because the asserting will have already all the needed semantic.